pkg/descriptions.json

```diff
@@ -30,6 +30,13 @@
       "Microsoft.CSharp.RuntimeBinder.CSharpBinderFlags"
     ]
   },
+  {
+    "Name": "Microsoft.Compatibility.HashCode",
+    "Description": "Provides the HashCode struct compatible with .NET Standard2.0. This package is not required when targetting .NET Standard2.1 or .NETCore App 3.0",
```

Comment:
```suggestion
    "Description": "Provides the HashCode type for .NET Standard 2.0. This package is not required beginning with .NET Standard 2.1 and .NET Core 3.0.",
```

/cc @mairaw

Reply:
Your suggestion LGTM. Minor nit would be that we usually say starting with instead of beginning with.

Reply:
> Minor nit would be that we usually say starting with instead of beginning with.

Thanks, reworded.